### ISSUE TITLE
TFT 32 beeps, no picture

### ISSUE THREAD

**Issue by Lino77** [makerbase-mks/MKS-TFT#106]
TFT 32 beeps, no picture whats that

**Comment by makerbase-mks** [comment]
Have you updated the firmware?

**Comment by Lino77** [comment]
Yes to the lautest Last week an fit worked but yesterday if i switched the Printer on they Beeps 
On the MKS sbase ans also on the mks Rumba+

**Comment by dudisr** [comment]
on 2.8 im having the same issue. I couldn't completed fix but instead of unzip then copy to the SD card i unzip directly in SD. This way mostly of the pictures appear, but still there are some shadows in screen or for ex. back button that never show its icon. Is usable but not fixed. Still waiting the guy to come up with a new version that show all  pictures.

**Comment by Lino77** [comment]
dont work only 
beep .... beep,beep...beep,beep...beep,beep...beep,beep...beep,beep...beep,beep...

**Comment by Lino77** [comment]
take a look 

https://www.youtube.com/watch?v=ljMkKY-kXWg

**Comment by Lino77** [comment]
@makerbase-mks what is now with my tft

**Comment by mks83** [comment]
Hi, I have a MKSTFT28 v3 that does exactly the same thing. Can someone please post the fix or verify that the board is defective. I have found a few threads like this but no one ever posts an outcome.

**Comment by karandex** [comment]
same problem here. it was working on the older firmware. it first showed some glitches on display. so decided to update it to old firmware and this started to happen. At least provide us with a compiled base bootloader so we can burn it via st link. you guys are very stubborn, you already have it closed source. at least help us fix out on our own.

**Comment by Popeyef5** [comment]
Same problem. Did anyone find a solution?

**Comment by ascarons22** [comment]
Yo brickee la mia a proposito para reparar este problema en otra tft,lo solucione con un stlink y reconfigurando el stm32,mi tft volvio a funcionar correctamente.

**Comment by dpatricio** [comment]
> Yo brickee la mia a proposito para reparar este problema en otra tft,lo solucione con un stlink y reconfigurando el stm32,mi tft volvio a funcionar correctamente.

tienes algún tutorial sobre esto?

**Comment by PrintEngineering** [comment]
Being that you guys have open-sourced the firmware now, is there any way you would be willing give some answers about this issue?
